### DISCUSSION TITLE
Production server

### DISCUSSION THREAD

**Discussion by tiagosimoesdev** [spatie/ray#47] [Q&A] [answered]
Is there any way to debug in a production server from my windows machine ?

**Comment by freekmurze** [comment, marked as answer]
This is on our roadmap.

We're adding the ability to connect to remote server in the coming months.

**Comment by pokmot** [comment]
You should be able to create an SSH tunnel that opens a port on your production server and terminate it to Ray on your local machine. But at present I presume you could not use "production" mode - unless there is an easy way to force Ray to still work in production?

**Reply by freekmurze** [reply]
In a Laravel app that runs in production, you could set the enable option in your config to `true` or call `ray()->enable()` somewhere early on in the request
